### DISCUSSION TITLE
Cannot run on CPU

### DISCUSSION THREAD

**Discussion by Tortoise17** [lllyasviel/Fooocus#2554] [Q&A] [answered]
Dear friends, I have worked with this tool successfully on GPU based workstation. But when I try to install this and after successful installation on laptop without GPU, it doesnt work and crashes in the middle. Although I have flagged --always-cpu. Is there any mistake I am making?

RAM available is 16 GB.
Free disc space 50 GB
Linux OS
No-GPU available on laptop.

Please guide and help.

**Comment by Rogala** [comment, marked as answer]
[Minimal System Memory 32GB](https://github.com/lllyasviel/Fooocus?tab=readme-ov-file#minimal-requirement)

**Comment by Tortoise17** [comment]
THank you so much. I will try on 32 GB RAM. Its already a great help.
